src/backend/mpi/mpi.cpp

```diff
@@ -4,14 +4,15 @@
  * @copyright Eta Scale AB. Licensed under the Eta Scale Open Source License. See the LICENSE file for details.
  */
 
-#include "../backend.hpp"
+#include <mpi.h>
```

Comment:
Please add the `// C headers` comment above this one. Here I actually got confused for a minute.
At first reading I thought this was ordered first because the name `mpi.h` matches `mpi.cpp`, but it is just the system header for the MPI C interface, not one of ours.

Reply:
Done